sympy/simplify/radsimp.py

```diff
@@ -555,6 +555,27 @@ def collect_const(expr, *vars, **kwargs):
     >>> collect_const(2*x - 2*y - 2*z, -2)
     2*x - 2*(y + z)
 
+    ---->> According to documentation, we have the second argument as *vars.
```

Comment:
The issue was to document what the various keyword argument accepted are, not to document what *args and **kwargs do. It is not necessary to document core Python features in every function. It is assumed that anyone using SymPy knows how to use Python.